<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QDialog" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Клиент</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lbData">
        <property name="text">
         <string>Дата</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="lePhone"/>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="btnDel">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="leLogin"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbLogin">
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbPhone">
        <property name="text">
         <string>Номер</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLineEdit" name="lePassword"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="leFIO"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbPassword">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbBirth">
        <property name="text">
         <string>рождения</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2" colspan="2">
       <widget class="QLineEdit" name="leData"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbFIO">
        <property name="text">
         <string>ФИО</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnSearch">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLineEdit" name="leSearch"/>
   </item>
   <item row="0" column="5" colspan="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>351</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="9">
    <widget class="QPushButton" name="btnSelectAll">
     <property name="text">
      <string>Выбрать Все</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2" colspan="8">
    <widget class="QTableWidget" name="twClient"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnEXit">
     <property name="text">
      <string>На главную</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnSchet">
     <property name="text">
      <string>Счёт</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnCLientKredit">
     <property name="text">
      <string>Перейти на</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>105</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="lbText1">
     <property name="text">
      <string>Кредиты</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="btnClientVklad">
     <property name="text">
      <string>Перейти на</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>270</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="7">
    <widget class="QPushButton" name="btnConnect">
     <property name="text">
      <string>Подключение</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>33</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="9">
    <widget class="QLabel" name="lbText2">
     <property name="text">
      <string>Вклады</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QTableWidget" name="twClientKredit"/>
   </item>
   <item row="4" column="5" colspan="5">
    <widget class="QTableWidget" name="twClientVklad"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
